xml/System.Security.Cryptography.Pkcs/SignerInfo.xml

```diff
@@ -59,9 +59,21 @@
         <Parameter Name="asnEncodedData" Type="System.Security.Cryptography.AsnEncodedData" Index="0" FrameworkAlternate="dotnet-plat-ext-3.0" />
       </Parameters>
       <Docs>
-        <param name="asnEncodedData">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="asnEncodedData">The attribute to add, an object with Abstract Syntax Notation One (ASN.1)-encoded data.</param>
+        <summary>Adds the specified attribute to the current instance's attribute collection. </summary>
```

Comment:
`unsigned attribute collection` (there is a signed and unsigned collection in the class so we need to be more specific)

Reply:
other comment might be that it is added `to document` - `current instance's attribute collection` sound kinda awkward but it actually represents the doc

Reply:
So you're saying this would be better, @krwq?:
```suggestion
        <summary>Adds the specified attribute to the current document.</summary>
```